testsuite/integration-tests/src/test/resources/META-INF/services/org.junit.platform.launcher.TestExecutionListener

```diff
@@ -1,2 +1 @@
 cz.xtf.junit5.listeners.TestExecutionLogger
-cz.xtf.junit5.listeners.ProjectCreator
```

Comment:
@fabiobrz I didn't understand why we had to introduce it here ... I believed openshift project creation was already taken care ....

Reply:
Thanks for your question @tommaso-borgato 
I removed the class from the SPI registration because we can only use it for tests that are expected to run on OpenShift.

Previously we had it in here because we knew we were running only on OpenShift, and project creation was expected to happen anyway.

This is why we let them implement the two different interfaces for project (OCP) or namespace (k8s) creation.